### DISCUSSION TITLE
Potential startup animation?

### DISCUSSION THREAD

**Discussion by 676575626572** [jayofelony/pwnagotchi#133] [Ideas]
I'm thinking it would be cool if we could get something like the OG Tamagotchi egg animation for first boot, so you know it's working. It would be nice if it was on the left, with a PSA on the left that says something like:

`Pwnagotchi is booting for the first time.`

`**Do NOT unplug.**`

`May take 5-20 minutes.`

Maybe delete the files after first boot so that it doesn't take up much space.

**Comment by rai68** [comment]
It does say that on first boot
